File: Sample.m
// Objective-C Sample Code
#import <Foundation/Foundation.h>

void greet(NSString *name) {
    NSLog(@"Hello, %@!", name);
}

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        greet(@"World");
    }
    return 0;
}

File: sample.ts
// TypeScript Sample Code
interface User {
  name: string;
  age: number;
}

const greet = (user: User): void => {
  console.log(`Hello, ${user.name}! You are ${user.age} years old.`);
};

const user: User = { name: "Alice", age: 30 };
greet(user);
